.changeset/thick-deers-act.md

```diff
@@ -0,0 +1,6 @@
+---
+'@itwin/itwinui-react': minor
+---
+
+Adding the ability to have custom `props` for each node of 'ColorPicker'
+New props are: `colorFieldProps`, `colorDotProps`, `panelLabelProps`, `inputContainerProps`, `inputFieldProps`, `labelProps` and `paletteContainerProps`.
```

Comment:
This changeset could be worded and formatted better. These prop names are meaningless unless we also mention which components they apply to.

You can also use markdown features like lists to make it easier to parse. For example, one bullet point per subcomponent.